content/en/docs/kubernetes/operator/automatic.md

```diff
@@ -300,6 +300,27 @@ spec:
         value: http,nestjs-core # comma-separated list of the instrumentation package names without the `@opentelemetry/instrumentation-` prefix.
 ```
 
+To keep all default libraries and disable only specific instrumentation
+libraries you can use the `OTEL_NODE_DISABLED_INSTRUMENTATIONS` environment
+variable. For details, see
+[Excluding instrumentation libraries](/docs/zero-code/js/configuration/#excluding-instrumentation-libraries).
+
+```yaml
+apiVersion: opentelemetry.io/v1alpha1
+kind: Instrumentation
+# ... other fields skipped from this example
+spec:
+  # ... other fields skipped from this example
+  nodejs:
+    env:
+      - name: OTEL_NODE_DISABLED_INSTRUMENTATIONS
+        value: fs,grpc # comma-separated list of the instrumentation package names without the `@opentelemetry/instrumentation-` prefix.
+```
+
+> Note that `OTEL_NODE_ENABLED_INSTRUMENTATIONS` and
+> `OTEL_NODE_DISABLED_INSTRUMENTATIONS` are mutually exclusive. If both are set,
+> all instrumentations will be disabled.
```

Comment:
```suggestion
> all instrumentations are disabled.
```

Reply:
Ah I pushed a commit changing the note type so it didn't let me commit this suggestion directly 😅  Made this change though, thanks for the explanation!